admin/tests.ini.mako

```diff
@@ -15,9 +15,9 @@ pyramid.available_languages = fr en
 pyramid.includes =
     pyramid_debugtoolbar
 
-sqlalchemy.url = postgresql://www-data:www-data@localhost/c2cgeoportal_tests
+sqlalchemy.url = postgresql://${dbuser}:${dbpassword}@${dbhost}:${dbport}/${db}
 schema = main
-srid = 3857
+srid = 21781
```

Comment:
I suppose this could come from vars too ?

Reply:
If we want to change that we will do it in an other pull request :-)